cpp/pybind/io/class_io.cpp

```diff
@@ -194,22 +194,65 @@ void pybind_class_io(py::module &m_io) {
     docstring::FunctionDocInject(m_io, "read_point_cloud",
                                  map_shared_argument_docstrings);
 
+    m_io.def(
+            "read_point_cloud_from_bytes",
```

Comment:
Please add python unit tests as well. You may put it in `python/test/io`. This will show the user how to prepare `py::bytes` in python and test the pybind functions.

Reply:
Done in 1984a20